CreateRestaurantFunction.cs

```diff
@@ -18,28 +18,29 @@ public static class CreateRestaurantFunction
         {
             log.Info("CreateRestaurant function processed a request.");
 
-            string strMongoDBAtlasUri = System.Environment.GetEnvironmentVariable("MongoDBAtlasURI");
-            log.Info($"Atlas connection string is { strMongoDBAtlasUri}");
+            var strMongoDBAtlasUri = System.Environment.GetEnvironmentVariable("MongoDBAtlasURI");
+            log.Info($"Atlas connection string is {strMongoDBAtlasUri}");
 
-            MongoUrl mongoUrl = new MongoUrl(strMongoDBAtlasUri);
+            var mongoUrl = new MongoUrl(strMongoDBAtlasUri);
             var settings = MongoClientSettings.FromUrl(mongoUrl);
             //for more on why we're using ServerSelectionTimeout, read https://scalegrid.io/blog/understanding-mongodb-client-timeout-options/
-            settings.ServerSelectionTimeout = new System.TimeSpan(0, 0, 5);
+            settings.ServerSelectionTimeout = TimeSpan.FromSeconds(5);
```

Comment:
This can be configured in the connection string? Is there are reason you are placing it here?